src/cgi/execve.cpp

```diff
@@ -0,0 +1,155 @@
+#include "../../inc/cgi.hpp"
+#include "../../inc/std98.hpp"
+#include "../../inc/utility.hpp"
+#include "cgi.hpp"
+#include "stdio.h"
+
+#define BUFFER_SIZE 4092
+
+CGI::CGI(void) : _pipefd(), _read_buffer() {}
+
+CGI::CGI(const CGI &src) : _pipefd(), _read_buffer() { *this = src; }
+
+CGI::CGI(const std::string &bin, const std::string &file, const std::string &query)
+	: _pipefd(), _read_buffer()
+{
+	_args.push_back(const_cast<char *>(bin.c_str()));
+	_args.push_back(const_cast<char *>(file.c_str()));
+	if (!query.empty())
+		_args.push_back(const_cast<char *>(query.c_str()));
+	_args.push_back(NULL);
+}
+
+/*
+ * Environment variables in PHP-CGI serve as a way to provide configuration values, system
+ * information, and sensitive data to PHP scripts during their execution, allowing for better separation of
+ * concerns, improved security, and easier management of dependencies and configurations.
+ */
+void
+CGI::set_env(char *args)
+{
+//		// Définit la route par laquelle tous les utilisateurs seront authentifiés (dans les applications
+//	 // prises
+//		// en charge).
+//		_env["AUTH_TYPE"] = "null";
+//		// Donne le nombre d'octets transmis par le client.
+//		_env["CONTENT_LENGTH"] = std98::to_string(1000);
```

Comment:
Why not directly give the number in the string ?
```suggestion
//		_env["CONTENT_LENGTH"] = "1000";
```